sql/2020/05_Media/image_mimetype_ext.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+# images mimetype vs extension
+SELECT
+  client,
+  COUNT(0) AS image_count,
+  ext,
+  mimetype
```

Comment:
Again would be good to see as percentage. What do you think of this:

```suggestion
  client,
  ext,
  mimetype,
  SUM(COUNT(0)) OVER (PARTITION BY client) AS total_images,
  COUNT(0) AS ext_mime_image_count,
  SAFE_DIVIDE(COUNT(0), SUM(COUNT(0)) OVER (PARTITION BY client)) AS total_image_pct,
  SAFE_DIVIDE(COUNT(0), SUM(COUNT(0)) OVER (PARTITION BY client, ext)) AS ext_pct,
  SAFE_DIVIDE(COUNT(0), SUM(COUNT(0)) OVER (PARTITION BY client, mimetype)) AS mime_pct
```